libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -779,6 +779,7 @@ Vector3f AP_AHRS::wind_estimate(void) const
         if (_sitl) {
             const auto &fdm = _sitl->state;
             wind = fdm.wind_ef;
+            wind.rotate_xy(radians(180));
```

Comment:
this should be fixed in the source where it is set in fdm, which backend gets this wrong?
changing it here would almost certainly break RealFlight, XPlane and others

Reply:
> this should be fixed in the source where it is set in fdm, which backend gets this wrong? changing it here would almost certainly break RealFlight, XPlane and others

The default simulation.  You can simulate it yourself - start a Plane sim normally, flip to backend type 10 and you will see the wind spin around 180-degrees.  I'll work up a patch to change it just in the affected backend.

Reply:
i think it is also worth bisecting this, I'm sure they were consistent in the past